vtex-apps/quickorder
# Fix subcategory not loading

#### What does this PR do? \*
Fixes the graphql query to load subcategories correctly 

#### How to test it? \*

On the category component, open the category and subcategories, the products should load correctly 

[Workspace](https://wender--dacris.myvtex.com/quickorder)
#### Describe alternatives you've considered, if any. \*



#### Related to / Depends on \*
